ADJ-Internship/WebApp/Views/_ViewStart.cshtml

```diff
@@ -1,3 +1,5 @@
 ﻿@{
-    Layout = "_Layout";
+    ViewData["Title"] = "Index";
+    Layout = "~/Views/Shared/_Layout.cshtml";
```

Comment:
Don't need to change anything in _ViewStart, just don't repeat the Layout declaration in other Views